docs/standard/serialization/system-text-json-how-to.md

```diff
@@ -182,6 +182,15 @@ To deserialize from a file by using asynchronous code, call the <xref:System.Tex
 
 :::code language="csharp" source="snippets/system-text-json-how-to/csharp/RoundtripToFileAsync.cs" id="Deserialize":::
 
+> [!TIP]
+> If you have JSON that you want to deserialize, and you don't have the class to deserialize it into, Visual Studio 2019 can automatically generate the class you need:
+>
+> 1. Copy the JSON that you need to deserialize.
+> 1. Create a class file and delete the template code.
+> 1. Choose **Edit** > **Paste Special** > **Paste JSON as Classes**.
+>
+> The result is a class that you can use for your deserialization target.
```

Comment:
Is there a Visual Studio link to this functionality at all? If so, you might want to include it here.

Reply:
There is nothing in the VS documentation about it, otherwise I would have linked to that and avoided listing the steps here. 😄